app/src/test/java/io/intrepid/contest/screens/contestcreation/categorieslist/CategoriesListPresenterTest.java

```diff
@@ -3,57 +3,44 @@
 import org.junit.Before;
 import org.junit.Test;
 import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
 
 import java.util.ArrayList;
 import java.util.List;
 
 import io.intrepid.contest.models.Category;
 import io.intrepid.contest.models.Contest;
-import io.intrepid.contest.testutils.BasePresenterTest;
+import io.intrepid.contest.testutils.TestPresenterConfiguration;
 
-import static org.mockito.ArgumentMatchers.any;
-import static org.mockito.ArgumentMatchers.argThat;
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
-public class CategoriesListPresenterTest extends BasePresenterTest<CategoriesListPresenter> {
+public class CategoriesListPresenterTest {
     @Mock
     CategoriesListContract.View mockView;
     @Mock
     Contest.Builder mockContestBuilder;
+    private CategoriesListContract.Presenter categoriesListPresenter;
```

Comment:
With `BasePresenterTest` you could assign it to `presenter` and keep using that variable in the test